mypyc/lib-rt/dict_ops.c

```diff
@@ -63,6 +63,18 @@ PyObject *CPyDict_Get(PyObject *dict, PyObject *key, PyObject *fallback) {
     return res;
 }
 
+PyObject *CPyDict_GetWithNone(PyObject *dict, PyObject *key) {
```

Comment:
What about just calling `CPyDict_Get` in the body with an extra `Py_None` argument? The extra call should have minimal performance impact, and this would reduce some code repetition?

Declare the function in `CPy.h`.